File: Modul 4/Soal 3/PRAK403-2410817210019-AchmadReihanAlfaiz.c
#include <stdio.h>

int main()
{
     int start, end;
     char symbol[] = "-";
     scanf("%d %d", &start, &end);

     int startStatic = start, endStatic = end;
     int range = (start >= end) ? start - end : end - start;

     for (int i = 0; i <= range; i++)
     {
          if (startStatic > endStatic)
          {
               printf("%d %d %s ", start, end, (i != range ? symbol : ""));
               start--;
               end++;
          }
          if (startStatic < endStatic)
          {
               printf("%d %d %s ", start, end, (i != range ? symbol : ""));
               start++;
               end--;
          }
     }

     return 0;
}

File: Modul 4/Soal 3/PRAK403-2410817210019-AchmadReihanAlfaiz.py
start, end = map(int, input().split())

start_static = start
end_static = end
range_val = abs(start - end)

for i in range(range_val + 1):
    if start_static > end_static:
        print(f"{start} {end}", end=(" - " if i != range_val else " "))
        start -= 1
        end += 1
    elif start_static < end_static:
        print(f"{start} {end}", end=(" - " if i != range_val else " "))
        start += 1
        end -= 1
